onnx/defs/tensor/defs.cc

```diff
@@ -114,7 +114,7 @@ Given an input of shape (N x C x D1 x D2 ... x Dn), output would be a 3-D tensor
 <br/>
 Where number of blocks extracted from each spatial dimension d is:
 ```
-num_blocks[d] = floor((input_spatial_shape[d] + 2 * padding[d] − dilation[d] * (kernel_size[d] − 1) − 1) / stride[d]) + 1
+num_blocks[d] = floor((input_spatial_shape[d] + 2 * padding[d] - dilation[d] * (kernel_size[d] - ) - 1) / stride[d]) + 1
```

Comment:
This change seems to lead to other changes in docs. Maybe by accident?

Reply:
No. the `-` in the master branch is not ascii code, so my windows anaconda throws when building ONNX.

Reply:
Maybe I missed something. Isn't this part of documentation? And I just don't quite understand what "(kernel_size[d] - )" means. But I am certainly okay if you believe this is the right way to describe the calculation.

Reply:
I will add `1` back. Thanks!